### PR TITLE
remove a duplicated lesson

### PR DESCRIPTION
**Pull request by gmauro**
I have merged the two lessons on Heterogeneous compute resources

### CODE REVIEW

File: topics/admin/tutorials/heterogeneous-compute/slides.html [hunk 1]
```diff
@@ -136,3 +136,7 @@
     * [https://github.com/galaxyproject/pulsar](https://github.com/galaxyproject/pulsar)
 * Pulsar Ansible
     * [https://github.com/galaxyproject/ansible-pulsar](https://github.com/galaxyproject/ansible-pulsar)
+
+# Exercise
```
Comment (erasche) [inline topics/admin/tutorials/heterogeneous-compute/slides.html]: Please remove this section, we don't want to link to outdated training resources.

File: topics/admin/tutorials/heterogeneous-compute/slides.html [hunk 2]
```diff
@@ -2,13 +2,13 @@
 layout: base_slides
 logo: "GTN"
 
-title: "Using Heterogeneous Compute Resources"
+title: "Running Jobs on Remote Resources with Pulsar"
```
Comment (erasche) [inline topics/admin/tutorials/heterogeneous-compute/slides.html]: While they can be remote systems they don't have to be.

I'd prefer to leave the original title / first section heading, but if you can think of a good way to say "heterogenous and/or remote", we can change to that?

Comment (gmauro) [reply topics/admin/tutorials/heterogeneous-compute/slides.html]: I think Pulsar should be on the title. What about "Using Pulsar to handle heterogeneous remote resources"?

Comment (gmauro) [reply topics/admin/tutorials/heterogeneous-compute/slides.html]: or better "Using Pulsar to handle heterogeneous compute resources"

Comment (erasche) [reply topics/admin/tutorials/heterogeneous-compute/slides.html]: > Using Pulsar to handle heterogeneous compute resources

sounds great!

File: topics/admin/tutorials/heterogeneous-compute/slides.html (hunk 1 above)
Comment (gmauro) [reply topics/admin/tutorials/heterogeneous-compute/slides.html]: ok

**Pull request merged**